src/widgets/infinite-hits/__tests__/infinite-hits-test.js

```diff
@@ -110,4 +110,38 @@ describe('infiniteHits()', () => {
     expect(helper.state.page).toBe(1);
     expect(helper.search).toHaveBeenCalledTimes(1);
   });
+
+  it('if it is the first page, then the props should contain isFirstPage true', () => {
+    const state = { page: 0 };
+    widget.render({
+      results: { ...results, page: 0, nbPages: 2 },
+      state,
+    });
+    widget.render({
+      results: { ...results, page: 1, nbPages: 2 },
+      state,
+    });
+
+    expect(render).toHaveBeenCalledTimes(2);
+    expect(render.mock.calls[0][0]).toMatchSnapshot();
```

Comment:
I know these tests were originally written like this, but we can make them more clear now. We shouldn't overuse snapshots. Asserting only the snapshot doesn't translate the intent of the test. We should add an expectation with the actual `isFirstPage` value.